session/bootstrap.go

```diff
@@ -535,7 +536,7 @@ func upgrade(s Session) {
 	}
 
 	if ver < version32 {
-		upgradeToVer29(s)
+		upgradeToVer32(s)
```

Comment:
could you add some test if it's possible?

Reply:
@zz-jason Do you mean to add a test for unused functions, or something else? I don't know how to test the cross-version issues on a specific version.